Scripts/azure_stack_provision.ps1

```diff
@@ -23,13 +23,16 @@ if ( [string]::IsNullOrWhiteSpace( $clone_lansa_version) ) {
 
  #####Parameters############
 $sql_server = "db-regression-$clone_lansa_version" #only accepting lower case
```

Comment:
This is the wrong Version. It should be `$lansa_version`. The ONLY place that `$clone_lansa_version` should be used is in retrieving the database backup to restore to the new database. So, change all the code accordingly.

Reply:
sure, Rob will change accordingly